composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.3.0",
         "fsi/datasource" : "~1.2",
-        "ruflin/elastica": "~1.2",
+        "ruflin/elastica": "~2.3",
```

Comment:
but its still bc break. Is there any plan to move it from "development" state to "ready to use".